setup.cfg

```diff
@@ -10,7 +10,7 @@ ignore = E402, E721
 max_line_length = 120
 
 [isort]
-known_third_party=dill,matplotlib,numpy,pkg_resources,pytest,setuptools,sklearn,torch,torchvision,trains
+known_third_party=dill,matplotlib,numpy,pkg_resources,pytest,requests,setuptools,sklearn,torch,torchvision,trains
```

Comment:
I think this file should not be modified

Reply:
That's pre-commit that modifies it

Reply:
I see. I'm not fan of adding it here... but OK.